tests/unit/backend/corpora/dataset_processing/test_process.py

```diff
@@ -558,3 +564,10 @@ def url(self):
         end = time.time()
         # check that tombstoning ends the download thread early
         self.assertLess(end - start, 11)
+
+    def test_format_slack_message(self):
+        dataset = self.generate_dataset(self.session)
+        os.environ["AWS_BATCH_JQ_NAME"] = "aws:single-cell:batch:us-west-2"
+        os.environ["AWS_BATCH_JOB_ID"] = "test_job_id"
+        message = format_slack_message(dataset.id)
+        json.loads(message)
```

Comment:
Do you want to make sure the message looks as expected?
Also it might be worth mocking requests.post and creating a failure to make sure the notification is triggered as expected

Reply:
I tested it out using [slacks message formatter](https://app.slack.com/block-kit-builder/T03N44GBB#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22New%20Paid%20Time%20Off%20request%20from%20%3Cexample.com%7CFred%20Enriquez%3E%5Cn%5Cn%3Chttps://example.com%7CView%20request%3E%22%7D%7D%5D%7D).